CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Added testing via CircleCI
+
+### Updated
+- Updated Tailwind dependency to 1.6.0
+- Removed animation plugin
```

Comment:
This is similar to how Craft CMS tracks their unreleased features.
https://github.com/craftcms/cms/blob/develop/CHANGELOG-v3.md

Reply:
Once a release is ready, we can just change the text to include the version number and date. Also, was your testing stuff ready?

Reply:
Yep it is, it's in `main` 👍